### ISSUE TITLE
evil-org

### ISSUE THREAD

**Issue by Bruno-Sosa** [thomasf/solarized-css#19]
The issue only happens when I have [evil-org](https://github.com/edwtjo/evil-org-mode) on.
I'm using spacemacs with the [org layer](http://spacemacs.org/layers/org/README.html).

The issue is:
when I export to html and open it in my web browser,
I can see everything before the first headline, but the rest is hidden.
None of the keybindings work `n/p m/X F/f` etc (I tried all to make sure),
and the HELP "toggle" doesn't appear, since the headlines don't show up.

The issue resolves itself when I remove the following line:
```org-mode
#+INFOJS_OPT: view:t toc:t ltoc:t mouse:underline buttons:0 path:http://thomasf.github.io/solarized-css/org-info.min.js
```
In other words, the CSS works just fine regardless of whether I'm in evil-org-mode or not.

(I just started with emacs, so I'm sorry if I'm wrong about something. Also really enjoying the project. Thank you everybody)

**Comment by Bruno-Sosa** [comment]
I created  one file with the bare minimum, and export to html twice.
Once with evil-org-mode, once without.

[tmp2.txt](https://github.com/thomasf/solarized-css/files/233509/tmp2.txt)
[tmp.txt](https://github.com/thomasf/solarized-css/files/233510/tmp.txt)

**Comment by Bruno-Sosa** [comment]
I manually tried to merge the difference
(and refreshing the pages in web-browser, by first renaming them to `tmp2.html` and `tmp.html`)
It seems that the problem is that the following line is missing in tmp2:

```html
<h1 class="title">tmp2</h1>
```
(line 212/213)
Which is strange since once that is included the table of contents starts to show

**Comment by Bruno-Sosa** [comment]
fixed! Since Solarized-CSS's `org-info.min.js` apperently requires a h1, for the rest to be recognized (ie h2, h3, etc) you can create one by setting the title property.
For example:
```org-mode
#+TITLE: Hello, and Welcome
```
